pkg/vpcmodel/connectivityAbstraction.go

```diff
@@ -152,8 +120,66 @@ func mergeConnectivityWithNodeSetAbstraction(
 		for _, conns := range nodeConns {
 			allConns = allConns.Union(conns)
 		}
-		res.updateAllowedConnsMap(src, nodeSet, allConns)
+		mergedConnectivity.updateAllowedConnsMap(src, nodeSet, allConns)
 	}
+	return mergedConnectivity
+}
 
-	return res
+// nodeSetAbstractionInformation() collects abstraction information of the nodeSet.
+// for now, it collects the connections that are missing for full abstraction.
```

Comment:
Please explain what is "missing connection"

Reply:
done